src/main/java/org/folio/spring/config/ApplicationConfig.java

```diff
@@ -13,12 +18,20 @@
   "org.folio.rest.validator",
   "org.folio.http",
   "org.folio.config.impl",
-  "org.folio.config.cache"})
+  "org.folio.config.cache",
+  "org.folio.rest.util.template"})
 public class ApplicationConfig {
   @Bean
   public PropertySourcesPlaceholderConfigurer placeholderConfigurer(){
     PropertySourcesPlaceholderConfigurer configurer = new PropertySourcesPlaceholderConfigurer();
     configurer.setLocation(new ClassPathResource("application.properties"));
     return configurer;
   }
+
+  @Bean
+  public ConversionService conversionService(List<Converter> converters) {
+    DefaultConversionService conversionService = new DefaultConversionService();
+    converters.forEach(conversionService::addConverter);
+    return conversionService;
+  }
```

Comment:
Would like to better understand why ConversionService is initialized here (with I assume all available converters injected)? As opposed to in TemplateFactory - Is this to guarantee a single instance of the class and converters?